gqlalchemy/installation.md

```diff
@@ -4,37 +4,53 @@ title: How to install GQLAlchemy
 sidebar_label: Installation
 ---
 
-There are two main ways of installing GQLAlchemy, through package managers such
+There are two main ways of installing GQLAlchemy: with package managers such
 as pip and Poetry, and by building it from source.
 
 ## Prerequisites
 
 To install GQLAlchemy, you will need the following:
-- **Python 3.7 - 3.10**
+
+- **Python 3.8 - 3.10**
 - GQLAlchemy is built on top of Memgraph's low-level Python client `pymgclient`, so you need to install `pymgclient` [build prerequisites](https://memgraph.github.io/pymgclient/introduction.html#build-prerequisites).
 
 :::caution
 GQLAlchemy can't be installed with Python 3.11 [(#203)](https://github.com/memgraph/gqlalchemy/issues/203) and on Windows with Python >= 3.10 [(#179)](https://github.com/memgraph/gqlalchemy/issues/179). If this is currently a blocker for you, please let us know by commenting on opened issues.
 :::
 
-## Install with Pip {#pip}
+## Install with pip {#pip}
 
-To install GQLAlchemy with pip, run the following command:
+Run the following command to install GQLAlchemy:
 
-```
+```bash
 pip install gqlalchemy
 ```
 
-## Build from source {#source}
+### Optional dependencies
+
+GQLAlchemy’s import/export capabilities aren’t included in the default installation.
+To make them available, use one of the following commands:
+
+```bash
+pip install gqlalchemy[arrow] # Support for the CSV, Parquet, ORC and IPC/Feather/Arrow formats
+pip install gqlalchemy[torch] # PyTorch support
+pip install gqlalchemy[dgl] # DGL support
 
-Clone or download the [GQLAlchemy source code](https://github.com/memgraph/gqlalchemy) locally and run the following command to build it from source with Poetry: 
+pip install gqlalchemy[ml] # Bundle of [torch] and [dgl]
 
+pip install gqlalchemy[all] # All of the above
 ```
-poetry install
+
+## Build from source
+
+Clone or download the [GQLAlchemy source code](https://github.com/memgraph/gqlalchemy) locally and run the following command to build it from source with Poetry:
+
+```bash
+poetry install --all-extras
```

Comment:
In that case, GQLAlchemy would be smoothly installed without the optional dependencies.
However, the next instruction is to run the unit tests, and the tests that use the optional dependency features would fail. Because of this, I’ve cut it short and instructed the reader to install it with `--all-extras`.

Reply:
I’ve added a few more sentences on `poetry install` with optional dependencies (the Poetry term is extras).